TowerForge/TowerForge/GameModule/GameModes/SurvivalGameMode.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SurvivalGameMode.swift
+//  TowerForge
+//
+//  Created by MacBook Pro on 09/04/24.
+//
+
+import Foundation
+import UIKit
+
+class SurvivalGameMode: GameMode {
+    var modeName: String = "Survival Game Mode"
+
+    var modeDescription: String = "Don't let the enemy invade your base!"
+
+    var gameProps: [any GameProp] = [PointProp(initialPoint: 0)]
```

Comment:
I think you need LifeProp here as well?

Reply:
LifeProp is only 1 live for survival mode so only one life though.

Reply:
But maybe yea it s better to let the gamers know, will add this